src/components/AccessibilityGuide/Errors.mdx

```diff
@@ -0,0 +1,99 @@
+import { Meta, Source } from "@storybook/blocks";
+
+import Link from "../Link/Link";
+
+<Meta title="Accessibility Guide/Errors" />
+
+# Errors
```

Comment:
What do you think about renaming this "UI Errors" or something more explicit that it's patron-facing errors? @bigfishdesign13 ?

Reply:
After our debate in standup, we are going to use `Error Messages` for the page heading. :)